src/hotspot/cpu/aarch64/c1_FpuStackSim_aarch64.cpp

```diff
@@ -28,3 +28,4 @@
 //--------------------------------------------------------
 
 // No FPU stack on AARCH64
+#include "precompiled.hpp"
```

Comment:
Why does Windows need this change?

Reply:
It seems that MSVC gives an [error](https://docs.microsoft.com/en-us/cpp/error-messages/compiler-errors-1/fatal-error-c1010?view=msvc-170) if a PCH specified on the command line is not included in the file. Since the use of precompiled headers is enabled by default, this compilation command gets the same PCH related arguments as the others. An alternative solution I found to be working is to add this file to the PCH exclude list here: [make/hotspot/lib/JvmOverrideFiles.gmk#L191](https://github.com/openjdk/jdk11u-dev/blob/6ec651fdc1c5ea0fe22731077446de1dd39cb24d/make/hotspot/lib/JvmOverrideFiles.gmk#L191), and then the include can be removed. Which one do you prefer?

Reply:
This change is OK, then. Let it stay.

Reply:
It would help avoid problems later if a comment was added to say that this include is needed to keep MSVC happy. Otherwise someone might delete it.